optional permissions, clickable icon

TheFantasticWarrior opened the pull request.

TheFantasticWarrior committed:
optional permissions, clickable icon

TheFantasticWarrior committed:
fix text alignment, add small icon to manifest

TheFantasticWarrior committed:
fix icon string in manifest, small changes

kuzn123 commented:
Hello! You have published this Imagus mod with the same name as the original extension... This, in my opinion, creates completely unnecessary confusion and misleads users about the name. 

I am the author of the rule-set of sieves (rules) for Imagus from the Ru-Board forum (at the moment this is the main rule-set). Therefore, I would like to ask you to rename your mod, for example, to "Imagus Mod" or the like, in order to avoid a COMPLETE match with the name of the original.

TheFantasticWarrior commented:
> I am the author of the rule-set of sieves (rules) for Imagus from the Ru-Board forum (at the moment this is the main rule-set). Therefore, I would like to ask you to rename your mod, for example, to "Imagus Mod" or the like, in order to avoid a COMPLETE match with the name of the original.

Hi, I am not the one that published the extension on chrome web store, I did publish on firefox with the name imagus-mod as you described https://addons.mozilla.org/en-US/firefox/addon/imagus-mod/, unless I got confused about your comment

kuzn123 commented:
Ok, thanks for the correction in the title.

Hm.. But in the Russian version, the name for some reason remained the same. And from the point of view of search convenience, it might be better to call "Imagus Mod" rather than "Imagus-mod".

Now:

Imagus-mod
https://addons.mozilla.org/en-US/firefox/addon/imagus-mod/

Imagus
https://addons.mozilla.org/ru/firefox/addon/imagus-mod/

TheFantasticWarrior committed:
custom download path
custom hotkeys
import/export config

TheFantasticWarrior committed:
add zh_tw and translation for 1 option in all lang

TheFantasticWarrior committed:
rename browser to chrome for chrome support

TheFantasticWarrior committed:
update readme

TheFantasticWarrior committed:
fix controls and other bugs

TheFantasticWarrior committed:
added css options, fix sieve update

TheFantasticWarrior committed:
fix css bug

TheFantasticWarrior committed:
fix color resetting on its own, max width/height

TheFantasticWarrior committed:
fix download

TheFantasticWarrior committed:
idk if downloads work in chrome so I changed a bit

TheFantasticWarrior committed:
actually fixed download(probably)

TheFantasticWarrior committed:
fix right click full zoom

TheFantasticWarrior committed:
fix download again

TheFantasticWarrior committed:
move download instructions to readme.md

TheFantasticWarrior committed:
fixing download on firefox

TheFantasticWarrior committed:
event.key->event.code for language support

TheFantasticWarrior committed:
fix show pop-up on key-press

I'm unable to prevent browser default behavior on alt key but image still shows
might be best to avoid alt key as show/hide key but I'll leave the option there

TheFantasticWarrior committed:
key->code fix for zoom in/out

TheFantasticWarrior committed:
fix chrome download

TheFantasticWarrior committed:
fix hotkeys

TheFantasticWarrior committed:
sieve: indicator to save, delete before update

TheFantasticWarrior committed:
update default sieve, fix open key and downloads

TheFantasticWarrior committed:
browser icon enhancement

tooltip change
shift/ctrl/whatever click the icon to turn on/off

TheFantasticWarrior committed:
added more file support, icon click change

TheFantasticWarrior committed:
chrome icon click now toggles

TheFantasticWarrior committed:
add right click icon, fix right click sieves

TheFantasticWarrior committed:
fix numpad key names, Equal renamed to Equal(Add)

TheFantasticWarrior committed:
add numpad toggle

TheFantasticWarrior committed:
fix cookie in incognito bug

TheFantasticWarrior committed:
bug fix

TheFantasticWarrior committed:
keep rules between update

TheFantasticWarrior committed:
Update README.md

TheFantasticWarrior committed:
custom save as file extension settings and more

keep sieve added simple version
fix bug that ignores delay before zooming

TheFantasticWarrior committed:
Merge branch 'new' of https://github.com/TheFantasticWarrior/chrome-extension-imagus into new

TheFantasticWarrior committed:
Multiple changes

removed old import/export TheFantasticWarrior/chrome-extension-imagus#24
tab key to change resolution https://www.reddit.com/r/imagus/comments/121oop7/comment/jquf22d/?utm_source=share&utm_medium=web2x&context=3

TheFantasticWarrior committed:
add delete key, fix exit fullzoom

TheFantasticWarrior committed:
wipe keep list when deleting sieves

TheFantasticWarrior committed:
crash fix

TheFantasticWarrior committed:
remove uncommon file extensions to fix twitter dl

TheFantasticWarrior committed:
keep adv keep checked, bug fix

keep advanced keeping checked on reload
fix https://www.reddit.com/r/imagus/comments/15bpyi6/comment/jts1iu9/?context=3
new update address

TheFantasticWarrior committed:
bug fix

fix import settings button
if you updated to v0.10.14.3 please re import the settings

TheFantasticWarrior committed:
bug fix

fix import settings button
if you updated to v0.10.14.3 please re import the settings

TheFantasticWarrior committed:
dont autosave on sieve update

TheFantasticWarrior committed:
fix #35

TheFantasticWarrior committed:
allow sieves assign download filename

just overwrite "this.CNT.filename", with or without file extension

TheFantasticWarrior committed:
update sieve update address

TheFantasticWarrior committed:
previous Equal->Equal(Add) change missed one spot

TheFantasticWarrior committed:
added grant hotkey
